netsim/ansible/templates/initial/linux/vanilla-ifconfig.j2

```diff
@@ -1,9 +1,13 @@
 {% macro ifconfig(intf) %}
-{% if intf.ipv4 is defined and intf.ipv4 is string %}
+{% if intf.ipv4 is defined %}
+{%   if intf.ipv4 is string %}
 set +e
 ip addr del {{ intf.ipv4 }} dev {{ intf.ifname }} 2>/dev/null
 set -e
 ip addr add {{ intf.ipv4 }} dev {{ intf.ifname }}
+{%   elif '_unnumbered_peer' in intf %}
+ip addr add {{ intf._parent_ipv4 }} peer {{ intf._unnumbered_peer }} dev {{ intf.ifname }}
```

Comment:
This would probably fail if you run "netlab initial" twice, but I'll fix that in another PR.

Reply:
I'll use **ip addr flush _intf_ scope global** instead.